operators/src/util/gdal.rs

```diff
@@ -122,6 +124,8 @@ pub fn raster_descriptor_from_dataset(
         spatial_reference: spatial_ref.into(),
         measurement: measurement_from_rasterband(dataset, band)?,
         no_data_value: rasterband.no_data_value(),
+        time: None,
+        bbox: None,
```

Comment:
We could compute at least the bbox… but should we?

Reply:
the bbox is for the whole (geo engine) dataset, so we would need to know all the files, open them and combine the boxes. Doesn't seem feasible.